app/main/views/services.py

```diff
@@ -150,6 +150,8 @@ def update_section(service_id, section_id):
             errors=errors
         )
 
+    flash(u"You\u2019ve edited your service. The changes are now live on the Digital Marketplace.", 'message')
```

Comment:
We generally flash a short message key from the view and have the long message text itself in the template file.  We've had extensive discussion about this in the past, and the reason behind doing it this way is that if the content of messages needs to be updated then the template is the natural place to look to do that, rather than in the backend view code.